### DISCUSSION TITLE
help installing

### DISCUSSION THREAD

**Discussion by jareddarkweb** [projectdiscovery/interactsh#233] [Q&A]
hi im trying to setup interactsh , On aws , running ubuntu 18 , and my domain name is on hostinger, 

when i try to install the go script i get this error
See https://golang.org/s/go15vendor for details.
root@ip-172-31-90-36:/home/ubuntu# go install -v github.com/projectdiscovery/interactsh/cmd/interactsh-server@latest
can't load package: package github.com/projectdiscovery/interactsh/cmd/interactsh-server@latest: cannot find package "github.com/projectdiscovery/interactsh/cmd/interactsh-server@latest" in any of:
        /usr/src/github.com/projectdiscovery/interactsh/cmd/interactsh-server@latest (from $GOROOT)
        /root/go/src/github.com/projectdiscovery/interactsh/cmd/interactsh-server@latest (from $GOPATH)
root@ip-172-31-90-36:/home/ubuntu#

**Comment by MetzinAround** [comment]
This sounds like a PATH issue. Make sure the directory where you installed interactsh is in $PATH.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
